app/models/dialog_import_validator.rb

```diff
@@ -10,6 +11,13 @@ def determine_validity(import_file_upload)
     raise ImportNonYamlError
   end
 
+  def determine_dialog_validity(dialog)
+    raise ParsedNonDialogError unless dialog['dialog_tabs']
+    check_dialog_tabs_for_validity(dialog['dialog_tabs'])
+  rescue ParsedNonDialogYamlError
```

Comment:
@abellotti removed the wrapper. I don't like the way this is right now (rescuing a ParsedNonDialogYamlError) and I think that this is a candidate for refactoring in the future (making them all throw ParsedNonDialogError) but I believe it's out of the scope of this PR.